omod/src/main/webapp/fragments/dashboardwidgets/labResults.gsp

```diff
@@ -0,0 +1,28 @@
+<div class="info-section lab-results-widget">
+    <div class="info-header">
+        <i class="${app.icon}"></i>
+        <h3>${ ui.message(app.label) }</h3>
+        <a href="${ ui.urlBind("/" + contextPath + "/" + detailsUrl, [ "patient.uuid": patient.uuid ]) }" class="right">
+            <i class="icon-share-alt edit-action" title="Edit"></i>
+        </a>
+    </div>
+    <div class="info-body">
+        <ul>
+        <% labResults.each{ labResult -> %>
+            <li>
+                <div style="display: block; overflow: hidden; padding-right: 5px; padding-bottom: 2px">
+                    <span style="float: left">
+                        ${ui.format(labResult.concept.getShortNameInLocale(ui.locale) != null ? labResult.concept.getShortNameInLocale(ui.locale) : labResult.concept.getPreferredName(ui.locale))}:&nbsp;
+                    </span>
+                    <span style="float: left">
+                        <strong>${ ui.format(labResult) }</strong>
+                    </span>
+                    <span style="float: right; font-size: small; color: #939393">
+                        ${ ui.formatDateWithClientTimezone(labResult.obsDatetime) }
```

Comment:
I believe we just want to use formatDatePretty here.  Are there other cases when we use the "withClientTimezone" functionality directly?

In the formatDatePretty, method you will see that it only uses the client time zone if the time zone version GP is set to true (which we have as false in our implementation)

I hate timezomes, but the explanation is that in our servers, the server timezone always maps to the clinic timezone, and that is the date we want to display.  ie, if the lab result came in on May 21, 2023 in the clinic timezone, that's the date we want to display, not the day before if per chance the client is in a different timezone.  All this "clientTimezone" stuff was added by Mekom because they generally keep their servers in UTC, regardless of the location of the clinic.